lib/config/options/index.ts

```diff
@@ -2034,6 +2034,12 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'automergeCommitMessage',
```

Comment:
```suggestion
    name: 'automergeMergeCommitMessage',
```

Let's make it more explicit (this change will need to be propagated elsewhere)